src/Pickles/Pickles.Test/ObjectModel/Json/TableRowToJsonTableRowMapperTests.cs

```diff
@@ -63,30 +67,56 @@ public void Map_TableRowWithRows_ReturnsObjectWithStrings()
         {
             var tableRow = new TableRow("first string", "second string", "third string");
 
-            var mapper = CreateMapper();
+            var testResult = new JsonTestResult
+            {
+                WasExecuted = false,
+                WasSuccessful = false
+            };
 
+            var mapper = CreateTableRowMapper();
             var actual = mapper.Map(tableRow);
 
-            Check.That(actual).ContainsExactly("first string", "second string", "third string");
+            Check.That(actual).Contains("first string", "second string", "third string");
+            Check.That(actual.Result.WasSuccessful).Equals(testResult.WasSuccessful);
```

Comment:
I prefer to have more explicit values in test cases - I think it improves readability. So I'd rather write .Equals(false), like in line 123.

The same comment applies to lines 81, 99 and 100 as well.

Reply:
absolutely - good idea